src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3056,16 +3053,16 @@ private static bool AttributeTypeNamesMatch(Type attributeType, Type reflectedAt
                 if (eventSourceAttrib != null && eventSourceAttrib.LocalizationResources != null)
                     resources = new ResourceManager(eventSourceAttrib.LocalizationResources, eventSourceType.Assembly);
 
-                if (source is not null)
-                {
+                manifest = (source == null)
```

Comment:
Can you simplify this? A nested conditional operator is hard to read and be sure of what it is doing without having to think about it. It looks like the logic is correct but I would like to see it easier to read.

Reply:
I will refactor it.